### PR TITLE
Update Module versions in distro

### PR DESCRIPTION
**Pull request by HerbertYiga**
Update Module versions in distro  cc @sherrif10  @mozzy11 @dkayiwa

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,46 +30,46 @@
...
-		<allergyuiVersion>1.8.5-SNAPSHOT</allergyuiVersion>
-		<formentryappVersion>1.4.3-SNAPSHOT</formentryappVersion>
+		<allergyuiVersion>1.8.4</allergyuiVersion>
+		<formentryappVersion>1.4.2</formentryappVersion>
 		<atlasVersion>2.2.5-SNAPSHOT</atlasVersion>
-		<appointmentschedulingVersion>1.15.0-SNAPSHOT</appointmentschedulingVersion>
-		<appointmentschedulinguiVersion>1.11.0-SNAPSHOT</appointmentschedulinguiVersion>
-		<namephoneticsVersion>1.18.0-SNAPSHOT</namephoneticsVersion> <!-- The omod for namephonetics is not included in the distro, but we keep the version around since it's a semi-supported add-on -->
-		<adminuiVersion>1.6.0-SNAPSHOT</adminuiVersion>
+		<appointmentschedulingVersion>1.14.0</appointmentschedulingVersion>
```
Comment (dkayiwa) [inline pom.xml:62]: This was a release with no changes since the previous one (1.13.0)

File: pom.xml [hunk 2]
```diff
@@ -30,46 +30,46 @@
...
+		<adminuiVersion>1.5.0</adminuiVersion>
 		<uitestframeworkVersion>2.4.0-SNAPSHOT</uitestframeworkVersion>
-		<legacyuiVersion>1.8.4-SNAPSHOT</legacyuiVersion>
-		<reportingcompatibilityVersion>2.0.9-SNAPSHOT</reportingcompatibilityVersion>
+		<legacyuiVersion>1.8.3</legacyuiVersion>
+		<reportingcompatibilityVersion>2.0.8</reportingcompatibilityVersion>
 		<fhirVersion>1.3.0-SNAPSHOT</fhirVersion>
 		<owaVersion>1.13.0-SNAPSHOT</owaVersion>
 		<reportinguiVersion>1.8.0-SNAPSHOT</reportinguiVersion>
-		<addresshierarchyVersion>2.15.0-SNAPSHOT</addresshierarchyVersion>
```
Comment (sherrif10) [inline pom.xml:72]: why not 2.15.0 , any reason why you reverted to 2.14.2

Comment (dkayiwa) [reply pom.xml:72]: https://wiki.openmrs.org/display/docs/Versioning

Comment (sherrif10) [reply pom.xml:72]: thanks @dkayiwa

**Pull request merged**